### DISCUSSION TITLE
How to fix "All checks have failed"

### DISCUSSION THREAD

**Discussion by mumershaikh44** [community/community#29264] [Pull Requests]
Hi,
We are facing an issue when we are trying to create pull request then it says "All checks have failed". Can anyone please help me how to fix this issue? I attached a screenshot for reference.
![github](https://user-images.githubusercontent.com/66939624/184339748-962e5b50-06d9-4e8c-ab60-eb1c795de2c0.PNG)

**Comment by queenofcorgis** [comment]
@mumershaikh44 👋🏼 

You can see which check has failed by using the tab within the Pull Request called checks to get a better idea. I hope that helps!

**Reply by mumershaikh44** [reply]
Under checks its showing "There are no checks for this commit". Please review screenshot.
![github-checks](https://user-images.githubusercontent.com/66939624/185065038-be849a14-cd7d-4423-9876-205f981d8704.PNG)

**Reply by queenofcorgis** [reply]
Are you able to share a link to the pull request so we can take a closer look?
